plugins/modules/azure_rm_vmskus.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2024
+# Nir Argaman <nargaman@redhat.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: azure_rm_vmskus
+
+version_added: "2.4.0"
+
+short_description: Get compute-related SKUs list
+
+description:
+    - Get details for compute-related resource SKUs.
+
+options:
+    location:
+        description:
+            - A region supported by current subscription.
+        type: str
+    resource_type:
+        description:
+            - Resource types e.g. "availabilitySets", "snapshots", "disks", etc.
+        type: str
+    size:
+        description:
+            - Size name, partial name is accepted.
+        type: str
+    zone:
+        description:
+            - Show skus supporting availability zones.
+        type: bool
+        default: False
+
+extends_documentation_fragment:
+    - azure.azcollection.azure
+
+author:
+    - Nir Argaman (@nirarg)
+
+'''
+
+EXAMPLES = '''
+- name: Gather Resource Group info
+  azure.azcollection.azure_rm_resourcegroup_info:
+    name: "{{ resource_group }}"
+  register: rg_info
+
+- name: List available VM SKUs
+  azure.azcollection.azure_rm_vmskus:
+    location: "{{ rg_info.resourcegroups.0.location }}"
+    resource_type: "virtualMachines"
+    size: "standard_B1"
+    zone: True
```

Comment:
Truthy value should be one of [false, true]
```suggestion
    zone: true
```

Reply:
Are you sure?
I see that "True" is also used in other places (and not true)
Python use True/False

Reply:
Ok, I see it came from "ansible-lint"
Probably need to fix also in other modules

Reply:
yes, I am sure!

Reply:
Fixed